cmd/mattermost/commands/export.go

```diff
@@ -202,25 +206,31 @@ func bulkExportCmdF(command *cobra.Command, args []string) error {
 		return errors.New("Nothing to export. Please specify the --all-teams flag to export all teams.")
 	}
 
+	attachments, err := command.Flags().GetBool("attachments")
+	if err != nil {
+		return errors.Wrap(err, "attachments flag error")
+	}
```

Comment:
nit: these checks are not necessary, as `cobra` would only complain if you `GetType` on a flag that has been set with a different type